content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -954,6 +954,7 @@ const user = await prisma.user.update({
 #### Remarks
 
 - To perform arithmetic operations on update (add, subtract, multiply, divide), use [atomic updates](#atomic-number-operations) to prevent race conditions.
+- If two or more upsert operations happen at the same time and the record doesn't already exist, then a race condition might happen. As a result, one or more of the upsert operations might throw a unique key constraint error. Your application code can trap this error and retry the operation. [Learn more](#unique-key-constraint-errors-on-upserts).
```

Comment:
Minor point: we don't seem to use 'trap this error' elsewhere in the docs. I'm more familiar with 'catch this error' which we do use, does this still work?

Reply:
Actually I think I prefer @zachtil's 'handle this error' above

Reply:
OK - I think "trap" is probably the terminology back from when I learned to code, back in the days of difference engines. I'll change it.